modules/qtwidgets/include/modules/qtwidgets/inviwoqtutils.h

```diff
@@ -225,6 +225,19 @@ IVW_MODULE_QTWIDGETS_API QSize emToPx(const QWidget* w, QSizeF);
 IVW_MODULE_QTWIDGETS_API int emToPx(const QWidget* w, double em);
 IVW_MODULE_QTWIDGETS_API int emToPx(const QFontMetrics& m, double em);
 
+
+
+/**
+ * Workaround for closing Widget using the "X" button in the titlebar
+ * This filter intercepts the CloseEvent and ignores it, and then hides the widget
+ */
+struct IVW_MODULE_QTWIDGETS_API WidgetCloseEventFilter : QObject {
```

Comment:
Would call this HideOnCloseEventFilter or HideInsteadOfCloseEventFilter

Reply:
At least I added a comment ;)